### PR TITLE
Filipino Translation

### PR DESCRIPTION
**Pull request by vinzruzell**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -36,6 +36,7 @@ def set_locale
     @locales = [
       { name: t('languages.en'), locale: :en },
       { name: t('languages.es'), locale: :es },
+      { name: t('languages.es'), locale: :fl },
```
Comment (Seunadex) [inline app/controllers/application_controller.rb]: @vinzruzell I noticed two `t('languages.es)` here. kindly double check

File: config/locales/devise.en.yml [hunk 2]
```diff
@@ -1,63 +0,0 @@
-# if me: DO NOT modify this file. if you need to add translations, add them to en.yml
-# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
-
```
Comment (Seunadex) [inline config/locales/devise.en.yml]: @vinzruzell any reason why all these were deleted?

**Comment by vinzruzell** [comment]
Hi. Still have some issue in here. I have already combined all commits. Thank you.

File: config/locales/devise.en.yml (hunk 2 above)
Comment (vinzruzell) [reply config/locales/devise.en.yml]: what u mean sir?

Comment (vinzruzell) [reply config/locales/devise.en.yml]: Ahh okay I thats the problem. I have just copied it from my first commit.

**Comment by cartothemax** [comment]
Hello @vinzruzell the tests are failing because the file https://github.com/ifmeorg/ifme/pull/831/files/acd5fe96e10d30b799185b1021ee7ab7b8f252d6#diff-f3d6e5d1f2709fef60571f46916a5f23 is deleted.

**Comment by julianguyen** [comment]
Just checking in @vinzruzell, any updates on this? :)

**Comment by julianguyen** [comment]
Closing this PR, preserving these commits and resolving merge conflicts in `fl-translations`

**Pull request closed without merging**